solutions/project/project-experimentation/network/subnet.yaml

```diff
@@ -110,6 +115,11 @@ spec:
     aggregationInterval: INTERVAL_5_SEC
     flowSampling: 0.5
     metadata: INCLUDE_ALL_METADATA
+  secondaryIpRange:
+    - ipCidrRange: 10.3.3.0/16 # kpt-set: ${gke-services-cidr-nane2}
+      rangeName: servicesrange # kpt-set: ${gke-services-range-name-nane2}
+    - ipCidrRange: 10.3.4.0/16 # kpt-set: ${gke-pod-cidr-nane2}
```

Comment:
this cidr range is invalid unless you change it to a /24

Reply:
fixed